py3status/docstrings.py

```diff
@@ -111,7 +111,7 @@ def create_readme(data):
 re_from_item = re.compile('^\s+-')
 re_from_data = re.compile('^@(author|license|source)($|[ \t])')
 re_from_tag = re.compile('((`[^`]*`)|[<>&])')
-re_from_defaults = re.compile('(\(default.*\))')
+re_from_defaults = re.compile('(\(default.*\))$')
```

Comment:
> We should get a proper parser going

Definitely. Because you're still missing the regexp with a tuple inside a tuple, and the one with a tuple inside a tuple inside a tuple, and so on.
Ideally, we could increment a counter whenever we find a `(` outside a string, decrement it whenever we find a `)` outside a string and check if the counter equals 0 a the end, but apparently python `re` doesn't support that.